apps/api/internal/flowcli/output.go

```
@@ -0,0 +1,560 @@
+			assets += *count
+		}
+		_, _ = fmt.Fprintf(table, "%s\t%s\t%s\t%d\t%s\t%d\t%s\n", item.ID, item.Type, item.Status, item.Revision, archived, assets, quoteHumanText(item.WorkingTitle))
+		return nil
+	})
+	if err != nil {
+		return err
+	}
+	return table.Flush()
+}
+
+func decodeListResponse(raw []byte) (listResponse, error) {
```

golangci-lint (2.12.2)

[error] func decodeListResponse is unused

(unused)